src/pyhf/utils.py

```diff
@@ -76,6 +76,44 @@ def loglambdav(pars, data, pdf):
     return -2 * pdf.logpdf(pars, data)
 
 
+def pll_and_estimators(mu, data, pdf, init_pars, par_bounds):
+    r"""The profile likelihood ratio :math:`\lambda(\mu)` for creating the various
+    test statistics, as defined in Equation (7) in `arXiv:1007.1727`_ .
+
+    .. _`arXiv:1007.1727`: https://arxiv.org/abs/1007.1727
+
+    .. math::
+
+       \lambda(\mu) = \frac{L(\mu,\hat{\hat{\pmb{\theta}}})}{L(\hat{\mu},\hat{\pmb{\theta}})}
+
+    Args:
+        mu (Number or Tensor): The signal strength parameter
+        data (Tensor): The data to be considered
+        pdf (|pyhf.pdf.Model|_): The HistFactory statistical model used in the likelihood ratio calculation
+        init_pars (Tensor): The initial parameters :math:`\pmb{\theta}`
+        par_bounds(Tensor): The bounds on the paramter values
+
+    .. |pyhf.pdf.Model| replace:: ``pyhf.pdf.Model``
+    .. _pyhf.pdf.Model: https://diana-hep.org/pyhf/_generated/pyhf.pdf.Model.html
+
+    Returns:
+        Tuple of (Tensor, Tensor, Float): The parameters :math:`\hat{\hat{\pmb{\theta}}}` that maximise the likelihood for the given value of :math:`\mu`, the parameters :math:`\hat{\pmb{\theta}}` that maximise the likelihood unconditionally (including :math:`\hat{\mu}`) and the value of :math:`-2\ln\lambda(\mu)`
```

Comment:
is the third argument really a float and not a tensor? I don't think `loglambdav` bails out of tensorland

Reply:
Indeed it is a Tensor - fixed that docstring now. Thanks!